hashes/govmomi

```diff
@@ -1,2 +1,2 @@
-# https://github.com/vmware/govmomi/archive/v0.30.2.tar.gz#/govmomi-0.30.2.tar.gz
-SHA512 (govmomi-0.30.2.tar.gz) = 19128ee73138ee8528678e577a6c03296a8782eaf5828cc1674bd42f793f0adb0121a1a38e039eeb130f89264d4ee31229cb61c5d6210199f92ad95b6f1e4661
+# https://github.com/vmware/govmomi/archive/v0.46.3/govmomi-0.46.3.tar.gz
```

Comment:
Changing the format here allow this SDK to be built with `UPSTREAM_SOURCE_FALLBACK` if the version is not present in the lookaside cache.